File: 944-Delete-Columns-to-Make-Sorted/944-Delete-Columns-to-Make-Sorted.java
class Solution {
    public int minDeletionSize(String[] strs) {
        boolean flag = false;
        int count = 0;
        for(int i = 0; i < strs[0].length() ; i++){
            for(int j = 1; j < strs.length; j++){
                if(strs[j-1].charAt(i) > strs[j].charAt(i)){
                    flag = true;
                    break;
                }
            }
            if(flag){
                count++;
                flag = false;
            }  
        }
        return count;
    }
}

File: 944-Delete-Columns-to-Make-Sorted/944-Delete-Columns-to-Make-Sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        n = len(strs)
        count = 0
        for i in range(len(strs[0])):
            string = []
            for j in range(n):
                string.append(strs[j][i])
            temp = sorted(string)
            print(string, "   ", temp)
            if temp != string:
                count += 1
        print(count)
        return count
